.github/CODEOWNERS

```diff
@@ -3,4 +3,4 @@
 #
 # For details see https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
-*  @dmitry-mightydevops @DanilaKazakevich @kseniyashaydurova @MikhailKorol-saritasa @owlfog @populov @roman-mightydevops @SergeyDeminSaritasa @udaltsovra @luciano-buono
+*  @dmitry-mightydevops @DanilaKazakevich @kseniyashaydurova @MikhailKorol-saritasa @populov @roman-mightydevops @SergeyDeminSaritasa @udaltsovra @luciano-buono @rtest12
```

Comment:
+ Nadya + Daria

Reply:
@dmitry-mightydevops 
Done

Reply:
@dmitry-mightydevops 
Nadezhda needs to add permission
![Screenshot from 2023-09-19 22-38-16](https://github.com/saritasa-nest/saritasa-devops-helm-charts/assets/93929010/b0754f24-0232-40a4-988e-59bd2ad5a670)

Reply:
@dmitry-mightydevops 
Just in case, can't merge this pr because "The base branch restricts merging to authorized users."

Reply:
Added Nadya to writers